dtests/src/test/java/io/snappydata/hydra/tpch/tpch.conf

```diff
@@ -4,14 +4,14 @@ INCLUDE $JTESTS/io/snappydata/hydra/northwind/startDualModeCluster.conf;
 INITTASK     taskClass   = io.snappydata.hydra.cluster.SnappyTest taskMethod  = HydraTask_executeSnappyJob
   io.snappydata.hydra.cluster.SnappyPrms-jobClassNames = io.snappydata.benchmark.snappy.tpch.TableCreationJob
   io.snappydata.hydra.cluster.SnappyPrms-appPropsForJobServer = "dataLocation=${dataDir},Buckets_Order_Lineitem=${buckets_Order_Lineitem},Buckets_Cust_Part_PartSupp=${buckets_Cust_Part_PartSupp},Buckets_Nation_Region_Supp=${buckets_Nation_Region_Supp},Nation_Region_Supp_col=${Nation_Region_Supp_col},Redundancy=${Redundancy},Persistence=${Persistence},Persistence_Type=${Persistence_Type},NumberOfLoadStages=${NumberOfLoadStages},isParquet=${Parquet}"
-  io.snappydata.hydra.cluster.SnappyPrms-userAppJar = snappydata-cluster_2.11-0.8-tests.jar
+  io.snappydata.hydra.cluster.SnappyPrms-userAppJar = snappydata-cluster_2.11-1.0.0-tests.jar
```

Comment:
Provide jar name as snappydata-cluster*tests.jar. I have used WildcardFileFilter while searching jar file location in framework in order to avoid hard coding of jar version. In this way, you don't need to change the confs for every release with new version of jar.